src/fshtypes/Configuration.ts

```diff
@@ -112,12 +112,14 @@ export type Configuration = {
   // file, do not include this property and include a `menu.xml` file in input/includes.
   menu?: ConfigurationMenuItem[];
 
+  // NO LONGER SUPPORTED: Use of the history property is no longer supported.  Authors should manage
```

Comment:
Nevermind.  But see comment above about making the reason we keep this clear (in the comments) and removing a lot of the comments that talk about how this property works since... it doesn't work!

Reply:
Updated in b2e8d06